src/main/java/com/notably/model/person/Path.java

```diff
@@ -0,0 +1,30 @@
+package com.notably.model.person;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represent a path of a block.
```

Comment:
Nit: Let's use the `{@link Block}` notation, so that when our javadoc is generated there will be a hyperlink to the `Block` interface.

That is, write `Represents a path to a {@link Block}`

Reply:
Updated it in the latest commit, did not know we could hyperlink a class in javadocs . Thank for the info

Reply:
👍